vm/mx.vm/mx_vm.py

```diff
@@ -34,9 +34,10 @@
 import mx_vm_benchmark
 import mx_vm_gate
 
-import os
-from os.path import join, relpath
 
+import os, json, shutil
+from os.path import join, relpath, exists, abspath
+from argparse import ArgumentParser
 
 _suite = mx.suite('vm')
```

Comment:
I would put this in `sdk` as then all projects have access to it.

Reply:
Any good reason not to put it in mx itself?